pages/index.page.tsx

```diff
@@ -320,7 +320,7 @@ const Home = (props: any) => {
         <section className='w-full h-[300px] lg:h-[367px] bg-gradient-to-r from-primary from-1.95% to-endBlue clip-both'>
           <div className='lg:w-full mx-auto text-center mt-28 '>
             <h2 className='text-h3mobile lg:text-h3 text-white mb-6'>Start contributing to JSON Schema</h2>
-            <button className='w-[170px] h-[45px] mx-auto rounded border-2 bg-primary text-white font-semibold'><a href='https://github.com/json-schema-org#-contributing-to-json-schema'>Contribute</a></button>
+            <button className='w-[170px] h-[45px] mx-auto rounded border-2 bg-primary text-white font-semibold'><a href='/newsletter'>Subscribe</a></button>
```

Comment:
Wha we need here is to show the component itself, no a button to navigate to the newsletter page. We need here to remove that extra step. Make it is easy for the user to subscribe.

See my first comment regarding how to make the component use different fonts/colours depending on the context.

My expectation is to implement the same approach than asyncapi. They show component in the landing page with white fonts:
https://www.asyncapi.com/en
And they show the component with black fonts in the newsletter page: 
https://www.asyncapi.com/newsletter

Reply:
added.